client/state/comments/actions.js

```diff
@@ -56,21 +56,20 @@ export const requestCommentsList = query => ( {
 	query,
 } );
 
-/***
- * Creates a remove comment action for a siteId, postId, commentId
+/**
+ * Creates an action that permanently deletes a comment
```

Comment:
Did you try it? If it acts like a post it'll probably set it's status to trash instead.